OpenEXR/IlmImf/CMakeLists.txt

```diff
@@ -17,11 +23,13 @@ ADD_CUSTOM_COMMAND (
   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/b44ExpLogTable.h
   COMMAND ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/b44ExpLogTable > ${CMAKE_CURRENT_BINARY_DIR}/b44ExpLogTable.h
   DEPENDS b44ExpLogTable
+  WORKING_DIRECTORY ${RUNTIME_DIR}
 )
 
 ADD_EXECUTABLE ( dwaLookups
   dwaLookups.cpp
 )
+message(STATUS "Using IlmBase version ${ILMBASE_LIBSUFFIX}")
```

Comment:
If this message is here at all it should use ILMBASE_VERSION as the suffix can be empty

Reply:
debug code :]